tests/context_tests.rs

```diff
@@ -640,6 +636,216 @@ mod test_policy_or {
         // There should be no errors setting an Or for a TRIAL session
         context.policy_or(trial_session, digest_list).unwrap();
     }
+
+    #[test]
+    fn test_policy_locality() {
+        let mut context = create_ctx_without_session();
+        let trial_session = context
+            .start_auth_session(
+                ESYS_TR_NONE,
+                ESYS_TR_NONE,
+                &[],
+                TPM2_SE_TRIAL,
+                utils::TpmtSymDefBuilder::aes_256_cfb(),
+                TPM2_ALG_SHA256,
+            )
+            .unwrap();
+        let trial_session_attr = TpmaSessionBuilder::new()
+            .with_flag(TPMA_SESSION_DECRYPT)
+            .with_flag(TPMA_SESSION_ENCRYPT)
+            .build();
+        context
+            .tr_sess_set_attributes(trial_session, trial_session_attr)
+            .unwrap();
+
+        // There should be no errors setting an Or for a TRIAL session
+        context.policy_locality(trial_session, 3).unwrap();
```

Comment:
What does `3` mean here?

Reply:
That was just a random value for testing.
Technically, it means the non-extended locality `1 + 2` (since anything <33 is non-extended localities).